### PR TITLE
Fabo/release PR now adds PENDING.md to CHANGELOG.md

### PR DESCRIPTION
**Pull request by faboweb**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `PENDING.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by sabau** [comment]
Nice, we should also changer `tasks/changelog-changed-check.sh`

File: tasks/createReleasePR.js [hunk 1]
```diff
@@ -60,15 +60,16 @@ if (require.main === module) {
   cli({}, async () => {
     console.log(`Making release...`)
     const changeLog = fs.readFileSync(__dirname + `/../CHANGELOG.md`, `utf8`)
+    const pending = fs.readFileSync(__dirname + `/../PENDING.md`, `utf8`)
```
Comment (sabau) [inline tasks/createReleasePR.js]: just nice to have for compatibility:
- we should create paths using the OS agnostic path.join(__dirname, '..', 'PENDING.md')

**Comment by faboweb** [comment]
done @sabau

**Comment by faboweb** [comment]
done @sabau

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,32 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.0.0-beta.22] - 2019-03-12
```
Comment (jbibla) [inline CHANGELOG.md]: what's going on here?

Comment (faboweb) [reply CHANGELOG.md]: That's from the tests. I will fix it.

**Comment by fedekunze** [comment]
does this also cleans up (deletes the content from) the PENDING file afterwards?

**Comment by faboweb** [comment]
> does this also cleans up (deletes the content from) the PENDING file afterwards?

yes

**Pull request merged**